CHANGELOG.md

```diff
@@ -21,6 +21,10 @@
   [Darron Schall](https://github.com/darronschall)
   [#463](https://github.com/SwiftGen/SwiftGen/issues/463)
   [#464](https://github.com/SwiftGen/SwiftGen/pull/464)
+* Fonts: disable a warning in generated font files for projects with `conditional_returns_on_newlines` SwiftLint rule enabled.
```

Comment:
Please add 2 spaces at the end of that line (see other entries for inspiration). _That's a formatting trick in Markdown to make the next line be on its own line once rendered_